source/containers.tex

```diff
@@ -1507,11 +1507,7 @@
 
 \pnum
 The phrase ``equivalence of keys'' means the equivalence relation imposed by the
-comparison and
-\textit{not}
-the
-\tcode{operator==}
-on keys.
+comparison.
```

Comment:
Would it be a further improvement to say "imposed by the comparison object" not just "imposed by the comparison"?

Reply:
Fixed.